src/handler.ts

```diff
@@ -263,7 +263,10 @@ export const mapResponse = (
 			set.headers['content-type'] = 'text/plain;charset=utf8'
 
 			if (res) {
-				set.headers['content-length'] = (response as string).length
+				set.headers['content-length'] = Buffer.byteLength(
+					response as string,
+					'utf8'
+				)
```

Comment:
`Content-Length` is calculated wrong. UTF-8 characters can have 2 bytes. use `Buffer.byteLength` instead